app/Czech.php

```diff
@@ -87,6 +38,15 @@ class Czech implements ValidatorInterface
     ];
 
     /**
+     * @param string $cnfFile
+     * @throws MissingBankCodesFileFile
+     */
+    public function __construct($cnfFile = __DIR__ . '/../cnf/czech-bank-codes.csv')
+    {
+        $this->validBankCodes = $this->getBankCodes($cnfFile);
```

Comment:
It would be nice to have this initialisation lazy. Performing disk operations in constructor is not ideal.